src/modules/registration/components/Registration.js

```diff
@@ -110,10 +109,9 @@ export const Registration = () => {
                     )}
                 </form>
 
-                {/*Insert Login route here*/}
-                <button className={styles.returnToLoginButton}>
+                <a href="/login" className={styles.returnToLoginButton}>
```

Comment:
```suggestion
                <a href={loginPath} className={styles.returnToLoginButton}>
```
import loginPath and use that instead